compiler/optimizer/op.go

```diff
@@ -220,15 +220,15 @@ func fieldOf(e dag.Expr) field.Path {
 	return nil
 }
 
-func copyOps(ops []dag.Op) []dag.Op {
+func CopyOps(ops []dag.Op) []dag.Op {
```

Comment:
Nit: This isn't used in any other package.
```suggestion
func copyOps(ops []dag.Op) []dag.Op {
```